code/__DEFINES/colors.dm

```diff
@@ -0,0 +1,54 @@
+
+#define COLOR_INPUT_DISABLED "#F0F0F0"
```

Comment:
But why

Reply:
taken from TG, just helps readability its the color of the input bar when you're in hotkey mode